solr.cfg

```diff
@@ -0,0 +1,15 @@
+[buildout]
+# XXX s/solr/master/
+# extends = https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/solr-base.cfg
+extends = https://raw.githubusercontent.com/4teamwork/ftw-buildouts/solr/solr-base.cfg
+
+zcml-additional-fragments += ${solr-settings:zcml}
+
+
+[solr-settings]
+port = 1${buildout:deployment-number}30
+
+
+[supervisor]
+programs +=
+    15 solr (startsecs=5) ${buildout:bin-directory}/solr-instance [fg] true ${buildout:os-user}
```

Comment:
It's now possible to restart solr, while the zope instances are running. So there's no need to wait.